gulp-tasks/html.js

```diff
@@ -5,7 +5,6 @@ const htmlmin = require('gulp-htmlmin');
 function minify() {
   return htmlmin({
     caseSensitive: true,
-    collapseWhitespace: true,
```

Comment:
how is bigger gonna be a file without this?

Reply:
the same, I've checked